file_converter/Program.cs

```diff
@@ -2,94 +2,73 @@
 using System.Linq;
 
 namespace file_converter
-{   
+{
     class Program
     {
+        private static string[] SUPPORTED_FORMATS = { "txt", "csv", "json", "xml" };
```

Comment:
nit: It is better to leave some space between different logical sections in your class. In this case logical sections are class-level constants and `GetFileExtension` method. I suggest adding a single empty line between them.